.github/workflows/continous-integration.yml

```diff
@@ -1,8 +1,9 @@
 name: continuous_integration
 
 on:
+  workflow_dispatch:
   schedule:
-    - cron: '0 4 * * 5'
+    - cron: "0 4 * * 5"
   push:
     branches:
       - master
```

Comment:
```suggestion
      - master
      - develop
```

Reply:
@kidunot89 it looks like this workflow supports both `push` and `pull_request`, but they're in separate entries (each entry supports `master` and `develop` branch).

I tried using the array `[push, pull_request]` configuration to consolidate them, but my linter complained. Is there a different way to do this?